src/Spectre.libStatistics/Math.h

```diff
@@ -380,4 +380,30 @@ std::vector<DataType> build(size_t size, Generator generator)
     return result;
 }
 
+/// <summary>
+/// Calculates differences between adjacent elements of given container.
```

Comment:
Please add information about the source of all assumptions that should be taken into account w.r.t. this function, i.e. page with MATLAB specification of their `diff` function behaviour. It'll be crucial to know, that e.g. output size is input size - 1, while it's not so explicit in tests.

Reply:
Done